tests/exporter/graph/test_graph_crawler.py

```diff
@@ -42,9 +43,12 @@ def test_generate_complete_experiment_graph(self):
         ingest_client = self.ingest
         crawler = GraphCrawler(MetadataService(ingest_client))
 
-        test_assay_process = MetadataResource.from_dict(self.files.get_entity('processes', 'mock-assay-process'))
-        test_project = MetadataResource.from_dict(self.files.get_entity('projects', 'mock-project'))
+        test_assay_process = MetadataResource.from_dict(self.mock_files.get_entity('processes', 'mock-assay-process'))
+        test_project = MetadataResource.from_dict(self.mock_files.get_entity('projects', 'mock-project'))
         experiment_graph = crawler.generate_experiment_graph(test_assay_process, test_project)
 
-        self.assertEqual(len(experiment_graph.nodes.get_nodes()), 15)
-        self.assertEqual(len(experiment_graph.links.get_links()), 4)  # 3 process links and 1 supplementary files link
+        expected_links = self.mock_files.get_links_json()
+
+        self.assertEqual(len(experiment_graph.nodes.get_nodes()), 18)
```

Comment:
Where does this 18 come from?

Reply:
I've updated the assertions to check the actual uuid's from links.json. Please check.